### PR TITLE
Request money on Sandbox if balance zero. (bunq/tinker_php#14)

### PR DESCRIPTION
**Pull request by JordyHeemskerk**
#14

### CODE REVIEW

File: src/BunqLib.php [hunk 1]
```diff
@@ -19,6 +19,7 @@
 use bunq\Model\Generated\Object\Pointer;
 use bunq\Util\BunqEnumApiEnvironmentType;
 use bunq\Util\InstallationUtil;
+use Habitat\Environment\Environment;
```
Comment (OGKevin) [inline src/BunqLib.php]: This can be removed right ? :P

File: src/BunqLib.php [hunk 2]
```diff
@@ -85,6 +97,7 @@ public function __construct(BunqEnumApiEnvironmentType $bunqEnumApiEnvironmentTy
         $this->environment = $bunqEnumApiEnvironmentType;
         $this->setupContext();
         $this->setupCurrentUser();
+        $this->requestSpendingMoneyIfNeeded();
```
Comment (OGKevin) [inline src/BunqLib.php:100]: TODO: bunq/sdk_php#122

After sdk update.

File: src/BunqLib.php [hunk 3]
```diff
@@ -464,4 +477,22 @@ public function getCurrentUser()
     {
         return $this->user;
     }
+
+    /**
+     * Requests money if the balance of the primary MA is equal to or less than zero.
+     */
+    private function requestSpendingMoneyIfNeeded()
+    {
+        if (BunqEnumApiEnvironmentType::SANDBOX()->equals($this->environment)
```
Comment (OGKevin) [inline src/BunqLib.php]: This logic can be extracted in its own separate method.

**Review by OGKevin — CHANGES_REQUESTED** [review]
Some minor changes.

**Comment by JordyHeemskerk** [comment]
@OGKevin moving UserContext refresh to new issue. Implemented comments.

**Review by OGKevin — APPROVED** [review]

**Comment by OGKevin** [comment]
@andrederoos

File: src/BunqLib.php (hunk 1 above)
Comment (einfachdavid89) [reply src/BunqLib.php]: Ja

**Pull request merged**